.github/workflows/main.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: windows-latest
```

Comment:
Need windows because this still has FullFX

Reply:
Do you mean now, or next major bump?

Reply:
If this is a brand new provider it could be done now. Start the way we mean to go on.

Perhaps get the basic things merged first and then do that.